addons/website_sale/views/templates.xml

```diff
@@ -544,7 +544,7 @@
                                 </div>
                             </div>
                             <hr />
-                            <p class="text-muted">
+                            <p class="text-muted oe_structure" id="oe_structure_website_sale_product_1">
```

Comment:
Can't work with default content like that for now... but we can create the inherited view by hand to add the default content (already did that in forum once).